src/molecules/navbar.js

```diff
@@ -0,0 +1,35 @@
+import React, { Component } from "react";
+import ReactSearchBox from 'react-search-box'
+
+export default class App extends Component {
+  data = [
```

Comment:
missing variable definition, currently it will be a `var`, which would be a global variable, If I'm not mistaken.